### PR TITLE
feat(gatsby-theme-bodiless): Implements move page functionality (#1054)

### PR DESCRIPTION
**Pull request by guilhermegpessoa**
## Changes
- Add move icon in menu edit page
- Add functionality rename page

## Test Instructions
- https://github.com/johnsonandjohnson/Bodiless-JS/issues/1054

## Related Issues
- #1054

### CODE REVIEW

File: .github/workflows/build-test-lint-simple.yml [hunk 1]
```diff
@@ -1,62 +1,44 @@
 name: Build, Test, Lint for Pull Requests
```
Comment (hvanyo) [inline .github/workflows/build-test-lint-simple.yml:1]: I don't think you want to revert this file back to node 12..

File: packages/gatsby-theme-bodiless/src/dist/withMovePageButton.tsx [hunk 2]
```diff
@@ -75,29 +75,34 @@ const movePage = async ({ origin, destination, client } : any) => {
...
-    } catch (e) {
+    } catch (e: any) {
       return Promise.reject(new Error(e.message));
     }
 
-    const result = await handle(client.directoryChild(origin));
+    const hasChild = await handle(client.directoryChild(origin));
 
-    if (result.response && result.message === 'Success') {
+    console.log('result.response', hasChild.response);
```
Comment (hvanyo) [inline packages/gatsby-theme-bodiless/src/dist/withMovePageButton.tsx:84]: Remove console.logs before merging

File: packages/bodiless-backend/src/backend.js [hunk 3]
```diff
@@ -630,6 +632,28 @@ class Backend {
       });
   }
 
+  static removeFile(route) {
```
Comment (konst8) [inline packages/bodiless-backend/src/backend.js:635]: @guilhermegpessoa Can we unify the names of such methods with names in packages/bodiless-backend/src/page.js?
removeFile / deleteFile
directoryExists / checkDirectory
etc.

**Review by konst8 — DISMISSED** [review]

File: .github/workflows/build-test-lint-simple.yml (hunk 1 above)
Comment (guilhermegpessoa) [reply .github/workflows/build-test-lint-simple.yml]: @hvanyo indeed that wasn't supposed to happen, fixed it.

File: packages/gatsby-theme-bodiless/src/dist/withMovePageButton.tsx (hunk 2 above)
Comment (guilhermegpessoa) [reply packages/gatsby-theme-bodiless/src/dist/withMovePageButton.tsx]: @hvanyo console.logs removed.

File: packages/bodiless-backend/src/backend.js (hunk 3 above)
Comment (guilhermegpessoa) [reply packages/bodiless-backend/src/backend.js:635]: @konst8 the methods were renamed.

**Review by konst8 — APPROVED** [review]

**Comment by konst8** [comment]
Declined in favor of https://github.com/johnsonandjohnson/Bodiless-JS/pull/1252

**Pull request closed without merging**